src/core/cmap.js

```diff
@@ -970,31 +970,38 @@ var CMapFactory = (function CMapFactoryClosure() {
   }
 
   function createBuiltInCMap(name, builtInCMapParams) {
-    if (name === 'Identity-H') {
-      return new IdentityCMap(false, 2);
-    } else if (name === 'Identity-V') {
-      return new IdentityCMap(true, 2);
-    }
-    if (BUILT_IN_CMAPS.indexOf(name) === -1) {
-      error('Unknown cMap name: ' + name);
-    }
-    assert(builtInCMapParams, 'built-in cMap parameters are not provided');
+    return new Promise(function (resolve, reject) {
```

Comment:
I recommend use Promise.resolve and Promise.reject without creating a new Promise until XMLHttpRequest below